src/test/isolation2/sql/copy_interrupt.sql

```diff
@@ -0,0 +1,14 @@
+!\retcode gpconfig -c client_connection_check_interval -v 20s;
```

Comment:
Maybe we should cancel it at the end? Otherwise, all subsequent tests will use this value.

Reply:
Okay, I will set it to 0 at the end

Reply:
Done